pennylane/qaoa/cycle.py

```diff
@@ -342,13 +342,15 @@ def loss_hamiltonian(graph: Union[nx.Graph, rx.PyGraph, rx.PyDiGraph]) -> qml.op
     >>> for k, v in edge_weight_data.items():
             g[k[0]][k[1]]["weight"] = v
     >>> h = loss_hamiltonian(g)
-    >>> print(h)
-      (-0.6931471805599453) [Z0]
-    + (0.0) [Z1]
-    + (0.4054651081081644) [Z2]
-    + (0.6931471805599453) [Z3]
-    + (0.9162907318741551) [Z4]
-    + (1.0986122886681098) [Z5]
+    >>> h
+    (
+        -0.6931471805599453 * Z(0)
```

Comment:
Feels like the minus should be in the same column as the `+` (not a blocker for this PR)

Reply:
I think that is actually generally the case in our current repr, that the sign of the coefficients are ignored and we always have a "minus", else we would need to invetigate the numerical values in each

Reply:
```python
>>> op = qml.dot([-1]*5, [X(0) @ X(1) for _ in range(5)])
>>> op
(
    -1 * (X(0) @ X(1))
  + -1 * (X(0) @ X(1))
  + -1 * (X(0) @ X(1))
  + -1 * (X(0) @ X(1))
  + -1 * (X(0) @ X(1))
)
```

Reply:
Note that this was the same for `Hamiltonian`
```
  (-1) [X0 X1]
+ (-1) [X0 X1]
+ (-1) [X0 X1]
+ (-1) [X0 X1]
+ (-1) [X0 X1]
```